### PR TITLE
docs: update certificate lifetime user guide

### PR DESCRIPTION
**Pull request by razvan**
# Description

Part of: https://github.com/stackabletech/issues/issues/586

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes

```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] CRD documentation for all fields, following the [style guide](https://docs.stackable.tech/home/nightly/contributor/docs/style-guide).
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
- [ ] Changes need to be "offline" compatible
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] Code contains useful logging statements
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated. Follows the [style guide](https://docs.stackable.tech/home/nightly/contributor/docs/style-guide).
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
- [ ] [Roadmap](https://github.com/orgs/stackabletech/projects/25/views/1) has been updated
```

### CODE REVIEW

File: docs/modules/trino/pages/usage-guide/operations/graceful-shutdown.adoc [hunk 1]
```diff
@@ -56,36 +56,24 @@ noticed that all tasks are complete
...
-====
-As of SDP version `23.7`, the secret-operator issues TLS certificates with a lifetime of 24h.
-It also adds an annotation to the Pod, indicating it requires a restart 30 minutes before the certificate expires (23.5h hours in this case).
-Currently, this results in all Pod using HTTPS (both coordinator and workers in a typical setup) to be restarted every 23.5 hours.
+== Shutdown triggers
+
+A shutdown operation can be triggered by multiple factors but a particularly important one is the renewal of the TLS certificates.
+This is important because it's a recurring event at regular intervals in the lifecycle of a Trino cluster.
+It is also an automatic process initiated by the commons operator when the TLS certificates issued by the secret operator are about to expire.
+Historically the lifetimes of these certificates have changed but they are always kept short (between one and seven days) by default.
```
Comment (sbernauer) [inline docs/modules/trino/pages/usage-guide/operations/graceful-shutdown.adoc:64]: I *think* we never had more than 24 hours, but I might be wrong
```suggestion
Historically the lifetimes of these certificates have changed but they are always kept short (around one day) by default.
```

File: docs/modules/trino/pages/usage-guide/operations/graceful-shutdown.adoc [hunk 2]
```diff
@@ -56,36 +56,24 @@ noticed that all tasks are complete
...
 
-The TLS certificate lifetime can be configured using `podOverrides` by setting `secrets.stackable.tech/backend.autotls.cert.lifetime` on every secret-operator volume.
-One sample configuration could look like:
+NOTE: It is recommended to set an explicit lifetime that is appropriate for your situation as described in xref:concepts:operations/temporary_credentials_lifetime.adoc[].
+
+To set the certificate lifetime to a fortnight, thus having the Trino pods restarted every two weeks, use the example below:
```
Comment (sbernauer) [inline docs/modules/trino/pages/usage-guide/operations/graceful-shutdown.adoc:68]: TIL! At first I've read Fortnite and was what the heck? :D

**Review by sbernauer — CHANGES_REQUESTED** [review]
Unrelated, but can you please run `cargo update -p hashbrown`?

**Comment by razvan** [comment]
> Unrelated, but can you please run `cargo update -p hashbrown`?

```
$ cargo update -p hashbrown
error: There are multiple `hashbrown` packages in your project, and the specification `hashbrown` is ambiguous.
Please re-run this command with one of the following specifications:
  hashbrown@0.14.5
  hashbrown@0.15.0
```

File: docs/modules/trino/pages/usage-guide/operations/graceful-shutdown.adoc (hunk 1 above)
Comment (razvan) [reply docs/modules/trino/pages/usage-guide/operations/graceful-shutdown.adoc:64]: _now_ it's seven days by default, but I'll take it

File: docs/modules/trino/pages/usage-guide/operations/graceful-shutdown.adoc (hunk 2 above)
Comment (razvan) [reply docs/modules/trino/pages/usage-guide/operations/graceful-shutdown.adoc:68]: found the gamer :)

**Comment by sbernauer** [comment]
As the vulnerable version is 0.15.0, you can use `cargo update -p hashbrown@0.15.0` as the command suggests.
However, I'm a fan a of simply `cargo update` to pull in new crate versions in general :)

**Review by sbernauer — APPROVED** [review]
Many thanks!

**Comment by lfrancke** [comment]
Is this going to be in 25.3 or 24.11.1?

**Comment by sbernauer** [comment]
25.3

**Pull request merged**